pkg/cloudprovider/provider/alibaba/provider.go

```diff
@@ -250,7 +250,7 @@ func (p *provider) Cleanup(machine *v1alpha1.Machine, data *cloudprovidertypes.P
 			if err := data.Update(machine, func(updatedMachine *v1alpha1.Machine) {
 				updatedMachine.Finalizers = kuberneteshelper.RemoveFinalizer(updatedMachine.Finalizers, finalizerInstance)
 			}); err != nil {
-				return false, fmt.Errorf("failed updating machine %v finzaliers: %v", machine.Name, err)
+				return false, err
```

Comment:
why removing error wrapping?

Reply:
I added the wrapping by myself but when I took another look on all the update functions, not of them wrap it. so I removed it for consistency